debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java

```diff
@@ -129,8 +129,15 @@ public void processRecord(Record r){
             }
             String seqName = getFromColumnSequenceMap(schemaOrDbname, r.t.tableName, r.afterValueColumns.get(i));
             if (seqName != null){
-                Long columnValue = Long.valueOf(r.afterValueValues.get(i).toString());
-                sequenceMax.put(seqName, Math.max(sequenceMax.get(seqName), columnValue));
+                Long columnValue = null;
+                String value = r.afterValueValues.get(i).toString();
+                try {
+                    columnValue = Long.valueOf(value);
+                    sequenceMax.put(seqName, Math.max(sequenceMax.get(seqName), columnValue));
+                } catch (NumberFormatException e) {
+                    //Skipping the sequences that are not on Integer columns
```

Comment:
Can you add more context in the comment on the type of sequences and why they could fail with a NumberFormatException?